sdk/Lusid.Sdk.Tests/tutorials/Ibor/Holdings.cs

```diff
@@ -196,6 +196,11 @@ public void Set_Target_Holdings()
             Assert.That(holdings[3].InstrumentUid, Is.EqualTo(instrument3));
             Assert.That(holdings[3].Units, Is.EqualTo(100.0));
             Assert.That(holdings[3].Cost.Amount, Is.EqualTo(10300.0));
+
+            // Get all the holding adjustments in the portfolio
+            var holdingAdjustments = _transactionPortfoliosApi.ListHoldingsAdjustments(TestDataUtilities.TutorialScope, portfolioCode);
+
+            Assert.That(holdingAdjustments.Values.Select(ha => ha.EffectiveAt), Is.EquivalentTo(new [] { day1 }));
```

Comment:
I think it would be useful to add a comment to say what this is trying to demonstrate (given the intent of these is to be tutorials rather than tests)